src/test/java/tests/HowItWorksTest.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.HowItWorksPage;
+
+public class HowItWorksTest extends BaseTest {
+
+    HomePage homePage = new HomePage(driver);
+    HowItWorksPage page = new HowItWorksPage(driver);
```

Comment:
All variables should be private or protected.  And please don't initialize pages in the fields of the class, it's better initialize them in tests methods(for example in "BeforeTest" method). Also you don't have to create HowItWorkspage, because you could use chain like in "howItWorksPage" method.